projects/core/src/cart/store/actions/cart-entry.action.ts

```diff
@@ -36,13 +36,13 @@ export class CartAddEntrySuccess extends EntityProcessesDecrementAction {
     public payload: {
       userId: string;
       cartId: string;
-      productCode: string;
-      quantity: number;
-      deliveryModeChanged: boolean;
-      entry: OrderEntry;
-      quantityAdded: number;
-      statusCode: string;
-      statusMessage: string;
+      productCode?: string;
+      quantity?: number;
+      deliveryModeChanged?: boolean;
+      entry?: OrderEntry;
+      quantityAdded?: number;
+      statusCode?: string;
+      statusMessage?: string;
```

Comment:
For future reference:
We noticed that the `CartModification` model coming from backend can have all those properties present or absent, depending on the `?fields` in the cart occ endpoint configuration. None of those fields is mandatory for Spartacus to work (because we don't take those properties in account, we're just reloading whole cart afterwards).